bundles/org.openhab.ui/web/src/assets/i18n/common/de.json

```diff
@@ -0,0 +1,41 @@
+{
+  "dialogs.ok": "OK",
+  "dialogs.cancel": "Abbrechen",
+  "dialogs.back": "Zurück",
+  "dialogs.done": "OK",
+  "dialogs.close": "Schließen",
+  "dialogs.copy": "Kopieren",
+  "dialogs.delete": "Löschen",
+  "dialogs.search": "Suchen",
+  "dialogs.search.items": "Items Suchen",
+  "dialogs.search.things": "Things Suchen",
+  "dialogs.search.rules": "Rules Suchen",
+  "home.overview.title": "Übersicht",
+  "home.overview.tab": "Übersicht",
+  "home.overview.button.documentation": "Dokumentation",
+  "home.overview.button.tutorial": "Tutorial",
+  "home.locations.title": "Standorte",
+  "home.locations.tab": "Standorte",
+  "home.equipment.title": "Geräte",
+  "home.equipment.tab": "Geräte",
+  "home.properties.title": "Eigenschaften",
+  "home.properties.tab": "Eigenschaften",
+  "home.cards.close": "@:dialogs.close",
+  "home.cards.analyze": "Analysieren",
+  "home.cards.analyzeAll": "Alles analysieren",
+  "home.otherApps": "Andere Apps",
+  "home.editHome": "Homepage bearbeiten",
+  "home.pinToHome": "Pin zum Startbildschirm",
+  "home.exitToApp": "Zurück zur App",
+  "home.tip.otherApps": "Dieses Panel öffnen, um andere Webinterfaces zu starten",
+  "sidebar.noPages": "Keine Seiten",
+  "sidebar.administration": "Verwaltung",
+  "sidebar.settings": "Einstellungen",
+  "sidebar.developerTools": "Entwicklung",
```

Comment:
Entwickler Tools

Reply:
Nice, I had tried "Entwicklerwerkzeuge" but it didn't fit in the android theme... This works better.